backend-database-tech-it-easy.sql

```diff
@@ -0,0 +1,151 @@
+-------- CREATE TABLES --------
+
+CREATE TABLE Products (
```

Comment:
Dit script is nu eenmalig te runnen. Als je hem nogmaals zou willen runnen dan zal die vertellen dat alle tabellen al bestaan. Daarom gaat een create table meestal gepaard met een drop table. Om bovenstaande dan te voorkomen kan je voor iedere create table het volgende neerzetten:
Drop table if exists naam_vd_tabel;